tt_eager/tt_dnn/op_library/CMakeLists.txt

```diff
@@ -35,10 +35,6 @@ set(TT_DNN_SRCS
     ${CMAKE_CURRENT_SOURCE_DIR}/non_zero_indices/non_zero_indices_op.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/fill_rm/fill_rm_op.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/fully_connected/fully_connected_op.cpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/pool/average_pool.cpp
```

Comment:
<img width="338" alt="Screenshot 2024-07-01 at 5 01 17 PM" src="https://github.com/tenstorrent/tt-metal/assets/169092593/193b2bce-69ff-4276-abd6-c625d2818e54">

Removed files still tracked in git?

Reply:
I didn't understand. The file still exists in main as well.